addons/point_of_sale/views/pos_ticket_view.xml

```diff
@@ -0,0 +1,135 @@
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-12">
+                                <ul>
+                                    <t t-foreach="pos_order.lines" t-as="order_line">
+                                        <li><t t-out="int(order_line.qty)"/> <t t-out="order_line.full_product_name"/> for <t t-out="format_amount(env, order_line.price_subtotal_incl, order_line.currency_id)"/></li>
+                                    </t>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col-12">
+                                <strong>Amounting to:</strong> <t t-out="format_amount(env, pos_order.amount_total, pos_order.currency_id)"/>
+                            </div>
+                        </div>
+                        <div class="row mt-4">
+                            <div class="col-12">
+                                <h4>Billing address:</h4>
+                            </div>
+                        </div>
+                        <!-- if no address, button to the detail form that redirects here -->
+                        <t t-if="not partner_address">
+                            Your address is missing or incomplete. <br/>
+                            Please make sure to <a t-att-href="address_url">fill all relevant information</a> before continuing.
+                        </t>
+                        <t t-else="">
+                            <div class="row">
+                                <div class="col-12">
+                                    <t t-out="partner.name"/>
+                                    <t t-if="partner.company_name">
+                                        , <t t-out="partner.company_name"/>
+                                    </t><br />
+                                    <t t-if="partner.vat">
+                                        <t t-out="partner.vat" /><br />
+                                    </t>
+                                    <t t-out="partner_address"/> <a role="button" t-att-href="address_url" class="btn btn-sm btn-link"><i class="fa fa-pencil"/> Edit</a>
+                                </div>
+                            </div>
+                        </t>
+                        <t t-if="additional_requirements">
+                            <div class="row mt-4">
+                                <div class="col-12">
+                                    <h4>Additional required information:</h4>
+                                    <t t-foreach="additional_requirements" t-as="additional_requirement">
+                                        <div class="row">
+                                            <div class="col-12">
+                                                <t t-out="additional_requirement"/>
```

Comment:
Correct me if I'm wrong, this is the select element created by calling `_field_to_html`, right? If so, I'm thinking of an alternative like so:

1. Create a template that will render the select element and its options.
```xml
<template id="selection_field">
   // inside here, define the select element and accept data from "additional_requirement"
<template>
```

2. Redefine (and probably rename) _field_to_html to return an object describing the content of the selection_field.
```py
def _field_to_html(self, ...):
    return {
        // information that can be used by selection_field template
    }
```

3. t-call here the selection_field template passing the "additional_requirement".

This way you don't need to manually render the element using Markup in the python code.